QUESTIONS.md

```diff
@@ -3,3 +3,17 @@
 What questions can you come up with?
 
 ## Cohort Questions
+
+- How does hosting our website work? Good sites to host from/ buy domains?
+
+- Stephanie can't wrap her head around React. She's having an allergic Reaction!
+
+- Want to know more about middleware, is this where security is provided? Why does your app need a hat (helmet)?
```

Comment:
Small version: middleware is an express function that every request passes through before you route it to a specific place. Helmet scans that request and looks for certain kinds of bad stuff to protect you. So yes middleware CAN protect you but not all middleware is for this purpose.